### PR TITLE
RFC: Initial readme and task template

### PR DESCRIPTION
**Pull request by carlosgjs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+
+By working on these tasks, students have the opportunity to work with SSEC engineers, who will act as mentors, review the code and provide feedback.
+
+
+## Pre-requisites
+
+- Being a UW student
+- Familiarity with [GitHub](https://github.com):
+    - Having a github account
+    - Knowledge of basic operations, such as pulling/forking a repo, creating branches, making commits and pushing.
```
Comment (carlosgjs) [inline README.md:14]: Anyone have a good tutorial for basic Git operations we can point to?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+*   @uwcdc @vanitech @lsetiawan @carlosgjs @dacb
```
Comment (lsetiawan) [inline .github/CODEOWNERS:1]: This is cool! I didn't know about this github feature!

File: README.md (hunk 1 above)
Comment (lsetiawan) [reply README.md:14]: I like this resource for open source development: https://www.dataschool.io/how-to-contribute-on-github/

Comment (carlosgjs) [reply README.md:14]: Thanks! Added a link

File: TASK.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+A description of the work to perform. Add enough details for someone unfamiliar with the project to work on the task. For L2 and L3 tasks, breaking them up in 2-3 steps will be helpful.
+
+## Tags: 
+
+Apply appropriate tags:
+- Level: L1, L2, L3
+- Language: python, shell, typescript
+- Cloud: aws, azure, gcp
+
```
Comment (lsetiawan) [inline TASK.md]: Let's convert this to an issue template: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

File: .github/ISSUE_TEMPLATE/task.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+description: Create a CodeUW Task
+title: "[Code UW]: "
+labels: ["task", "codeuw"]
+body:
+  - type: input
+    id: contact
+    attributes:
+      label: Contact
+      description: Who can provide additional details about the task?
+      placeholder: ex. netit@uw.edu
```
Comment (lsetiawan) [inline .github/ISSUE_TEMPLATE/task.yml]:
```suggestion
      placeholder: ex. netid@uw.edu
```

File: .github/ISSUE_TEMPLATE/task.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: What needs to be done?
+      placeholder: |
+        A description of the work to perform. Add enough details for someone unfamiliar with the project 
+        to work on the task. For L2 and L3 tasks, breaking them up in 2-3 steps will be helpful.
```
Comment (lsetiawan) [inline .github/ISSUE_TEMPLATE/task.yml:21]:
```suggestion
        to work on the task. For L2 and L3 tasks, breaking them up in 2-3 steps will be helpful.
        If a task have existing issue, provide a link to it as well.
```

**Review by lsetiawan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/task.yml (hunk 5 above)
Comment (carlosgjs) [reply .github/ISSUE_TEMPLATE/task.yml:21]: Added a separate field in the form

**Pull request merged**